webpack.config.ts

```diff
@@ -1,11 +1,27 @@
+/// <reference types="./source/globals" />
+
 import path from 'path';
-import {readdirSync} from 'fs';
+import {readdirSync, readFileSync} from 'fs';
 import webpack from 'webpack';
+import SizePlugin from 'size-plugin';
 import TerserPlugin from 'terser-webpack-plugin';
 import CopyWebpackPlugin from 'copy-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
-// @ts-ignore
-import SizePlugin from 'size-plugin';
+
+function parseFeatureDetails(name: string): FeatureInfo {
+	const content = readFileSync(`source/features/${name}.tsx`, {encoding: 'utf-8'});
+	const rawRegex = ['description', 'screenshot', 'disabled']
+		.map(field => `\n\t${field}: '(?<${field}>[^\\n]+)'`) // Named group regex
+		.join('|');
+
+	const feature = new RegExp(rawRegex).exec(content)!.groups!;
+	feature.name = name;
+	return feature as unknown as FeatureInfo;
```

Comment:
The description field might have some `\'` that will be shown like that in the options page.

Technically we prefer `’` so this isn't a problem, but we don't enforce it automatically so some might slip through.

Help wanted to create a local `eslint` plugin to enforce and autofix it (replace `\'` with `’`)

Reply:
I don't have time to write a rule, but this is what you need:

```js
'use strict';

const create = context => {
	return {
		Literal: node => {
          	const {value} = node;
     
			if (typeof value !== 'string') {
				return;
			}
          
        	const fixedValue = value.replace(/'/, '‘');

			if (fixedValue === value) {
				return;
			}

			context.report({
				node,
				message: 'Use other quote kind',
				fix: fixer => fixer.replaceTextRange([node.range[0] + 1, node.range[1] - 1], fixedValue)
			});
		}
	};
};

module.exports = {
	create,
	meta: {
		type: 'problem',
		fixable: 'code'
	}
};
```

https://astexplorer.net/#/gist/3a67df351a57608ae38a6b55102376d6/c8425e326ddce75bf7242295c8818134ef2d2e16

Reply:
Maybe you should propose it on https://github.com/sindresorhus/eslint-plugin-unicorn